diff --git a/.gitignore b/.gitignore
index 8fa7ea45811fd626b161a4709415527bcd0938bb..637df17e32a5fb77a902cc75d73fe6a9aee7808c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -97,25 +97,29 @@ public
 
 # Storybook build outputs
 .out
 .storybook-out
 
 # Temporary folders
 tmp/
 temp/
 
 # Editor directories and files
 .vscode/*
 !.vscode/extensions.json
 .idea
 .DS_Store
 *.suo
 *.ntvs*
 *.njsproj
 *.sln
 *.sw?
 
 # OS generated files
 Thumbs.db
 ehthumbs.db
 Desktop.ini
 node_modules
+!backend/node_modules
+backend/node_modules/*
+!backend/node_modules/node-cache
+!backend/node_modules/node-cache/**
